src/WFS.js

```diff
@@ -72,6 +80,11 @@ L.WFS = L.FeatureGroup.extend({
     });
   },
 
+  setFilterBBox: function (bbox) {
```

Comment:
It will be separate function, something like Strategy in OpenLayers. In any case user can call loadFetures with specified filter.